app/models/comments.rb

```diff
@@ -0,0 +1,15 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  belongs_to :user
+
+  validates :text, presence: true
+
+  after_create :update_post_comment_counter
```

Comment:
- [ ] Good job so far but after_create is called only when a new record is created and saved to the database. This means that it will not be called when an existing record is updated.
after_save is called both when a new record is created and saved, and when an existing record is updated and saved.
In other words, after_create is triggered only once during the lifetime of an object, while after_save is triggered every time an object is saved, whether it is being created for the first time or updated. As a result, I would encourage you to use after_save since we always want to update.

Reply:
@Olamarx Done, thank you for this info I appreciate it.